### PR TITLE
save metrics

### PR DESCRIPTION
**Pull request by vilchik-elena**
Fixes SonarSource/SonarTS#97

### CODE REVIEW

File: its/projects/plugin/main.ts [hunk 1]
```diff
@@ -3,5 +3,9 @@ function foo(x: number): boolean {
     return true;
   }
 
+  if (x >= x) { // NOSONAR
```
Comment (pynicolas) [inline its/projects/plugin/main.ts]: If we want to test that NOSONAR has some impact, this line may have to be identical to line 2 except for NOSONAR.

File: its/src/test/java/org/sonar/typescript/its/TypescriptPluginTest.java [hunk 2]
```diff
@@ -59,29 +66,67 @@ public static void prepare() {
...
   }
 
   @Test
-  public void should_have_loaded_issues_into_project() {
+  public void should_have_loaded_issues_into_project_and_ignore_issue_with_nosonar() {
     SearchWsRequest request = new SearchWsRequest();
-    request.setProjectKeys(Collections.singletonList("SonarTS-plugin-test"));
-    assertThat(newWsClient().issues().search(request).getIssuesCount()).isGreaterThanOrEqualTo(1);
+    request.setProjectKeys(Collections.singletonList(PROJECT_KEY)).setRules(ImmutableList.of("typescript:no-identical-expressions"));
+    assertThat(newWsClient().issues().search(request).getIssuesCount()).isEqualTo(1);
```
Comment (pynicolas) [inline its/src/test/java/org/sonar/typescript/its/TypescriptPluginTest.java]: Specifying the expected line would make the test clearer and more robust.

File: its/src/test/java/org/sonar/typescript/its/TypescriptPluginTest.java [hunk 3]
```diff
@@ -59,29 +66,67 @@ public static void prepare() {
...
   }
 
-  public static SonarScanner createScanner() {
+  @Test
+  public void should_save_metrics() {
+    // Size
+    assertThat(getProjectMeasureAsDouble("ncloc")).isEqualTo(9);
+    assertThat(getProjectMeasureAsDouble("lines")).isEqualTo(12);
```
Comment (pynicolas) [inline its/src/test/java/org/sonar/typescript/its/TypescriptPluginTest.java]: lines, files and directories are computed by SQ. Should we test them?

Comment (vilchik-elena) [inline its/src/test/java/org/sonar/typescript/its/TypescriptPluginTest.java]: @pynicolas dropped everything done by SQ 94035dd424f99cf2923ff661b9bf5d0c3984529f

File: its/src/test/java/org/sonar/typescript/its/TypescriptPluginTest.java (hunk 2 above)
Comment (vilchik-elena) [inline its/src/test/java/org/sonar/typescript/its/TypescriptPluginTest.java]: @pynicolas 👍  94035dd424f99cf2923ff661b9bf5d0c3984529f

File: its/projects/plugin/main.ts (hunk 1 above)
Comment (vilchik-elena) [inline its/projects/plugin/main.ts]: @pynicolas Indeed!  94035dd424f99cf2923ff661b9bf5d0c3984529f

**Pull request merged**